extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -172,18 +172,28 @@ private function handleXmlParseError( $errors, $xmlMarkup ) {
 		return [ $renderedValue, 'markerType' => 'nowiki' ];
 	}
 
-	private function getThemeWithDefault( $params, PPFrame $frame ) {
-		$value = isset( $params[ 'theme-source' ] ) ? $frame->getArgument( $params[ 'theme-source' ] ) : false;
-		$themeName = $this->getThemeName( $params, $value );
+	private function getThemes( $params, PPFrame $frame ) {
+		$themes = [ ];
 
-		//make sure no whitespaces, prevents side effects
-		return Sanitizer::escapeClass( self::INFOBOX_THEME_PREFIX . preg_replace( '|\s+|s', '-', $themeName ) );
-	}
+		if ( isset( $params[ 'theme' ] ) ) {
+			$staticTheme = trim( $params[ 'theme' ] );
+			if ( !empty ( $staticTheme ) ) {
+				$themes[] = $staticTheme;
+			}
+		}
+		if ( !empty( $params[ 'theme-source' ] ) ) {
+			$variableTheme = trim( $frame->getArgument( $params[ 'theme-source' ] ) );
+			if ( !empty( $variableTheme ) ) {
```

Comment:
Decide if we put empty lines above `if` blocks or not, and keep it consistent. I like the new lines.